CONTRIBUTING.md

```diff
@@ -415,3 +415,62 @@ https://github.com/apache/incubator-superset/pull/3013
     git add .
     git commit -a -m "New doc version"
     git push origin master
+
+## Publishing a Pypi release
+
+  We create a branch that goes along each minor release `0.24`
+  and micro releases get corresponding tags as in `0.24.0`. Git history should
+  never be altered in release branches.
+  Bug fixes and security-related patches get cherry-picked 
+  (usually from master) as in `git cherry-pick -x {SHA}`.
+  
+  Following a set of cherries being picked, a release can be pushed to 
+  Pypi as follows:
+
+  .. code::
+
+    # branching off of master
+    git checkout -b 0.25
+
+    # cherry-picking a SHA
+    git cherry-pick -x f9d85bd2e1fd9bc233d19c76bed09467522b968a
+    # repeat with other SHAs, don't forget the -x
+
+    # source of thruth for release numbers live in package.json
+    vi superset/assets/package.json
+    # hard code release in file, commit to the release branch
+    git commit -a -m "0.25.0"
+
+    # create the release tag in the release branch
+    git tag 0.25.0
+    git push apache 0.25 --tags
+
+    # check travis to confirm the build succeeded as
+    # you shouldn't assume that a clean cherry will be clean
+    # when landing on a new sundae
+
+    # compile the JS, and push to pypi
+    ./pypi_push.sh
+
+    # publish an update to the CHANGELOG.md for the right version range
+    # looking the latest CHANGELOG entry for the second argument
+    ./gen_changelog.sh 0.25.0 0.22.1
```

Comment:
versions looks reversed, also as an example something like 0.24 0.25 would be easier to follow

Reply:
Following the logic in the underlying tool. I don't feel strongly about it either way. npm's github-changes

Reply:
From gen_changelog.sh:
```
# usage:  ./github-changes 0.20.0 0.20.1
# will overwrites the local CHANGELOG.md, somehow you need to merge it in
github-changes -o apache -r incubator-superset --token $GITHUB_TOKEN --between-tags $1...$2
```

So $1 should be older than $2

Reply:
Oh you're right!